packages/url_launcher/url_launcher_ios/example/ios/RunnerUITests/URLLauncherUITests.swift

```diff
@@ -28,10 +28,11 @@ class URLLauncherUITests: XCTestCase {
       button.tap()
       let webView = app.webViews.firstMatch
       XCTAssertTrue(webView.waitForExistence(timeout: 30.0))
-      XCTAssertTrue(app.buttons["ForwardButton"].waitForExistence(timeout: 30.0))
-      XCTAssertTrue(app.buttons["Share"].exists)
-      XCTAssertTrue(app.buttons["OpenInSafariButton"].exists)
-      let doneButton = app.buttons["Done"]
+      XCTAssertTrue(app.buttons["OpenInSafariButton"].waitForExistence(timeout: 30.0))
```

Comment:
Does the CI run this test on xcode 26, or I should manually verify this change?

Reply:
I tested this in a PR draft running Xcode 26 beta 5 https://github.com/flutter/packages/pull/9748. I rebased that branch onto this one. It failed before, but now passes:

```
Testing started
Successfully ran iOS xctest for packages/url_launcher/url_launcher_ios/example

[packages/url_launcher/url_launcher_ios completed in 1m 29s]
```
https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8706532896630430321/+/u/Run_package_tests/native_test/stdout